lapis2/src/main/kotlin/org/genspectrum/lapis/model/SiloFilterExpressionMapper.kt

```diff
@@ -308,7 +322,7 @@ class SiloFilterExpressionMapper(
         val value = extractSingleFilterValue(values, originalKey)
 
         try {
-            return value.toInt()
+            return value?.toInt()
```

Comment:
Does this also need a "is blank" handling?
`/sample/aggregated?ageFrom=` returns a bad request with `"ageFrom '' is not a valid integer: For input string: \"\""`, while a POST with `{ "age": null}` is ok.

Reply:
(Same for floats)